### PR TITLE
Introduce Site Package

### PR DESCRIPTION
**Pull request by shomix**
This PR will add support for working with the Web-App organizations, Projects and Deployments.
This project will:
- Increase the version of `@spheron/core` ( added new methods wrapping around the REST API ) 
- Create a new package called `@spheron/site`
  - Checkout the https://github.com/spheronFdn/docs/pull/23 for methods of site sdk 

Readme: https://github.com/spheronFdn/sdk/tree/SPH-1124/packages/site

### CODE REVIEW

**Comment by shomix** [comment]
@shomix need to increase the versions in the readme files

**Review by dusanstanis — DISMISSED** [review]
Looks pretty straightforward to me !

**Review by rekpero — DISMISSED** [review]
lgtm

File: packages/core/src/spheron-api/enums.ts [hunk 1]
```diff
@@ -79,6 +79,9 @@ enum AppTypeEnum {
 }
 
 enum ProviderEnum {
+  GITHUB = "GITHUB",
+  GITLAB = "GITLAB",
+  BITBUCKET = "GITLAB",
```
[the remark this answers is not in the document]
Comment (shomix) [reply packages/core/src/spheron-api/enums.ts]: Fixed

File: packages/core/src/spheron-api/index.ts [hunk 2]
```diff
@@ -340,6 +368,237 @@ class SpheronApi {
...
+    branch: string;
+    provider: ProviderEnum;
+    repositoryName: string;
+    root?: string;
+  }): Promise<{ suggestedFramework: FrameworkEnum }> {
+    const response = await this.sendApiRequest<{
+      suggestedFramework: FrameworkEnum;
+    }>(
+      HttpMethods.GET,
+      `/v1/deployment/framework/suggestion?owner=${options.owner}&branch=${options.branch}&provider=${options.provider}&repo=${options.repositoryName}&root=${options.root}`
```
[the remark this answers is not in the document]
Comment (shomix) [reply packages/core/src/spheron-api/index.ts]: Updated

**Review by ilij4 — APPROVED** [review]

**Review by rekpero — APPROVED** [review]

**Pull request merged**